bot/start.js

```diff
@@ -109,6 +109,41 @@ const askForConfirmation = async (user, command) => {
   }
 };
 
+const { SocksProxyAgent } = require('socks-proxy-agent');
```

Comment:
Why moving this code here? I think we should't touch `start.js` if we don't have a good reason

Reply:
Sorry for the ambiguity. I have updated PR and improved the commit msg to provide an explanation for the issue with integration tests failing